charts/rabbitmq/README.md

```diff
@@ -21,5 +21,14 @@ To install abbitmq chart from remote repository , use the regular helm install/u
 
 `helm install -f ./rabbitmq-values.yaml rabbitmq https://charts.bitnami.com/bitnami/rabbitmq-11.10.0.tgz`
 
+in this deployment, the Clustering and Service Account are enabled in order to make it HA and the replica count is set to 3.
 
+### Create router for rabbitmq-management console
 
+After installing RabbitMQ helm chart, manually create a new router in OCP for rabbitmq service with following config:
+
+Path: /
```

Comment:
This is good.  Consider adding a non-helmed manifest of the route and just referring to it with instructions on how to manually deploy it. The reason for not including it in the helm chart is so it doesn't automatically get installed as we don't normally want to have a route to the management as it's a potential security hole....esp since we've used non-secret passwords throughout.